src/plugins/vis_type_timelion/server/series_functions/es/lib/create_date_agg.js

```diff
@@ -6,9 +6,9 @@
  * Side Public License, v 1.
  */
 
-import _ from 'lodash';
 import { buildAggBody } from './agg_body';
 import { search } from '../../../../../../plugins/data/server';
+import { METRIC_TYPES } from '../../../../../data/common/search/aggs/metrics/index';
```

Comment:
wrong import.  Please replace to  '../../../../../data/server'

Reply:
Resolved